client/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "singleQuote": false,
+  "trailingComma": "es5",
```

Comment:
FWIW this is the new default for Prettier. It's what my laptop was using while I was away from home, so most files are already formatted with this comma style.

My desktop uses a slightly older version version of Prettier which defaults to never allowing trailing commas. I quiet like the ES5 style, but it's only a Prettier option and not a built-in `eslint` option for `comma-dangle` 😄